gateway/templates/default.conf.template

```diff
@@ -14,11 +14,6 @@ map $http_x_real_ip $forwarded_ip {
   default $remote_addr;
 }
 
-map $forwarded_proto $var_proxy_cookie_flags {
-  https   "JSESSIONID secure samesite=strict";
-  default "off";
-}
-
```

Comment:
You should retain this rule or else the JSESSIONID cookie won't be marked as `secure`.